features/FxA-51-connect-another-device/README.md

```diff
@@ -133,6 +133,107 @@ Second, for users that click on one of the two app store icons, we are unable to
 
 These two limitations force us to use the previously discussed proxy measurements to determine success.
 
+### Acceptance Criteria/Testing
+
+The feature is currently being A/B tested, only members of the "treatment" group see the new UI. For testing purposes, it is possible to force being a member of the "treatment" group by opening the verification link with the following additional query parameters:
+`&forceExperiment=connectAnotherDevice&forceExperimentGroup=treatment`
+
+The UI presented to the user changes depending on where the user verifies their email address. Testing of each of the possible combinations is needed.
+
+The convention used below is section headings indicate where the user signs up, section sub-headings indicate where the user opens the verification link.
+
+#### Signup in Firefox Desktop
+
+##### Verify in the same Firefox Desktop.
```

Comment:
@ryanfeeley suggests I change the form of the sub-heading to "When the user clicks the email verificaiton link which opens in Firefox for iOS:"